packages/manager/modules/vps/src/import/vps.service.js

```diff
@@ -31,6 +31,8 @@ export default /* @ngInject */ function VpsService(
 
   const vpsTabVeeamCache = $cacheFactory('UNIVERS_WEB_VPS_TABS_VEEAM');
 
+  const apiCatalogProductName = 'virtualprivateserver';
```

Comment:
Reminder: in production, we will have to use /order/catalog/public/vps

Reply:
I didn't know that. Thanks ^^

Reply:
we will handle it once all features has been validated by QA